### ISSUE TITLE
illegal_argument_exception","reason":"No endpoint or operation is available at [_ltr]"}]

### ISSUE THREAD

**Issue by sunshisonghit** [o19s/elasticsearch-learning-to-rank#117]
I had installed the plugins 
by using  _cat/plugins,it says : gkTOxcV ltr-query      0.1.2-es5.5.0
but when I use the _ltr : curl -XGET 172.16.11.153:28123/_ltr/
it occures the following exception:
"type":"illegal_argument_exception","reason":"No endpoint or operation is available at [_ltr]"}

HELP!!!!

**Comment by ebernhardson** [comment]
The older version of the plugin you have installed didn't yet have this endpoint. It's mostly an informational endpoint we added to have something there, but not important for actually training and using models. One option would be to use the 1.0RC2, but i see you have 5.5.0 and we've only released the plugin for 5.5.2.  Having to release the plugin for every possible version of elasticsearch is something we havn't completely figured out yet but will soon.

Besides this endpoint, is the reset of the plugin working or are there more issues?
